charts/quickwit/values.yaml

```diff
@@ -38,49 +38,55 @@ securityContext:
 
 # Additional global env
 environment: {}
-  # KEY: VALUE
+# KEY: VALUE
 environmentFrom: []
   # - secretRef:
   #     name: quickwit
   # - configMapRef:
-  #     name: quickwit
+#     name: quickwit
 
 configMaps: []
   # - name: configmap1
-  #   mountPath: /quickwit/configmaps/
+#   mountPath: /quickwit/configmaps/
```

Comment:
This comment shouldn't be changed as well

Reply:
Fixed